RELEASE_GUIDELINE.md

```diff
@@ -0,0 +1,40 @@
+# RELEASE GUIDELINES
+- There is a scheduled release on every 15th of the month on the following repositories:
+  - [Litmus](https://github.com/litmuschaos/litmus)
+  - [Chaos-Operator](https://github.com/litmuschaos/chaos-operator)
+  - [Chaos-Exporter](https://github.com/litmuschaos/chaos-exporter)
+
+- These repositories use release version according to the [Semantic Versioning](https://semver.org/)
+- These repositories create a new docker image tag after the release is done. Following are the docker images:
```

Comment:
Docker images with release tags are pushed upon creation of a github release. The CI configuration to enable this can be found <here>. (provide reference of the push script/travis/circle ci configs which perform this step)